cmake/Tools/common.py

```diff
@@ -552,7 +558,7 @@ def exec(self, arguments, capture_stdout=False, suppress_stderr=False, cwd=None)
                 call_args.append(str(arguments))
             logging.debug("exec(%s)", subprocess.list2cmdline(call_args))
             result = subprocess.run(call_args,
-                                    shell=True,
+                                    shell=(platform.system() == 'Windows'),
```

Comment:
Do we need 'shell' anymore?

Reply:
Windows might still require it but will confirm

Reply:
Confirmed, still required for Windows.